Mathlib/Data/Real/Sqrt.lean

```diff
@@ -113,6 +113,7 @@ theorem sqrt_div (x y : ℝ≥0) : sqrt (x / y) = sqrt x / sqrt y :=
   map_div₀ sqrtHom x y
 #align nnreal.sqrt_div NNReal.sqrt_div
 
+@[fun_prop]
 theorem continuous_sqrt : Continuous sqrt := sqrt.continuous
```

Comment:
```suggestion
@[continuity, fun_prop]
theorem continuous_sqrt : Continuous sqrt := sqrt.continuous
```
We should probably decide whether to standardize on `fun_prop` or `continuity`, but it'll be easier to refactor later if every relevant lemma is tagged with `@[continuity]` anyway.